### DISCUSSION TITLE
Bulleted list Question

### DISCUSSION THREAD

**Discussion by gjblajian** [dolanmiu/docx#1754] [Q&A] [answered]
@dolanmiu when you create a bulleted list, how do you indicate which bullet you want? I know all of the valid ones are predefined but I am fuzzy about how to tell the library which one I want my specific bullet entry to point to [circle, hollowCircle, square, hollowSquare, etc...].

https://docx.js.org/#/usage/bullet-points

![image](https://user-images.githubusercontent.com/5183939/199111407-3e185096-dfe9-4dd8-af16-03b132e1eee6.png)

**Comment by dolanmiu** [comment, marked as answer]
So it is pretty easy

You simply specify the unicode value, and it will simply work:

Look at this example here:

https://github.com/dolanmiu/docx/blob/master/demo/3-numbering-and-bullet-points.ts#L64

**Reply by danielschlitt** [reply]
How would I specify this when patching a document?
